content/blog/entries/cc-search-accessibility-week9-10/contents.lr

```diff
@@ -0,0 +1,131 @@
+title: Accessibility Improvements: Final Changes and Modal Accessilibity 
+---
+categories:
+cc-search
+community
+gsoc
+open-source
+
+---
+author: AyanChoudhary
+---
+series: gsoc-2020-ccsearch-accessibility
+---
+pub_date: 2020-08-12
+---
+body:
+
+These are the last two weeks of my internship with CC. I am working on improving the accessibility of cc-search and internationalizing it as well.
+This post contains details of my work done to make accessibility improvements to the search result page and the image detail page and also covers some advanced accessiblity improvement details.
+
+The topics included in this post cover:
+
+1. Tooltip accessibility and keyboard interactions
+2. Improve modal accessibility and implement trap focus
+3. Fix ```<label>``` for form elements
+
+The first stage involved fixing the license explanation tooltips. These tooltips worked fine on click but did not respond to keypress events.
+The solution to overcome this was to use an event listener on the element which would would execute the ```showLicenseExplanation``` function onClick.
+Luckily ```VueJS``` provides this function inbuilt via the ```v-on:keyup``` attribute. So after change the code looks as follows:
+
+```
```

Comment:
you can add `html` to your code blocks to get some syntax highlighting, put "html" after the opening "```" like this:

<img width="366" alt="Screen Shot 2020-08-31 at 12 17 07 PM" src="https://user-images.githubusercontent.com/6351754/91774175-ee54cf00-eb83-11ea-9ab5-ea04a5e3f4dc.png">

You can test if "vue" works, but it might not. I'm not sure if the blog's syntax highlighter has it built-in.

Reply:
@zackkrida  I have made the changes and "vue" does not work because we don't have the required alias I think.